content/source/docs/enterprise/admin/agents-on-tfe.html.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: "enterprise"
+page_title: "Terraform Cloud Agents on TFE - Infrastructure Administration - Terraform Enterprise"
+---
+
+# Terraform Cloud Agents on TFE
+
+Terraform Cloud Agents allow Terraform Enterprise to communicate with isolated,
+private, or on-premises infrastructure. By deploying lightweight agents within a
+specific network segment, you can establish a simple connection between your
+environment and Terraform Enterprise which allows for provisioning operations and
+management.
+
+Terraform Cloud Agents on TFE behave identically to [Terraform Cloud Agents on TFC](https://www.terraform.io/docs/cloud/agents/index.html). Please reference this
```

Comment:
Can this just be `/docs/cloud/agents/index.html`, since this is the same base domain?

Reply:
Everything else I had seen on other pages had been fully qualified links. I'll play with this though :+1:

Reply:
Oh yes - just to second @RussellRollins I think where possible we should do internal links to our own docs!